sections/main-product.liquid

```diff
@@ -1,729 +1,96 @@
-<section
-  id="MainProduct-{{ section.id }}"
-  class="page-width section-{{ section.id }}-padding"
-  data-section="{{ section.id }}"
->
-  {{ 'section-main-product.css' | asset_url | stylesheet_tag }}
-  {{ 'component-accordion.css' | asset_url | stylesheet_tag }}
-  {{ 'component-price.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rte.css' | asset_url | stylesheet_tag }}
-  {{ 'component-slider.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rating.css' | asset_url | stylesheet_tag }}
-  {{ 'component-loading-overlay.css' | asset_url | stylesheet_tag }}
-  {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+<div class="color-{{ section.settings.color_scheme }}">
```

Comment:
Another question is, should the product information be receiving the global settings from `content-container` ? 😅 

Something for designers to look into/let us know. 

Multicolumn, the text block from image with text, image banner, rich text, etc, are receiving it. I wonder if it should here as well 🤔